### PR TITLE
Localize commands3

### PR DESCRIPTION
**Pull request by JavierMatosD**
This PR localizes messages in the following files:
- commands.format-manifest.cpp
- commands.find.cpp
- command.civerifyversions.cpp
 
I also added the function append_nl() which simply appends a newline character to the localized message. Previously, newline characters were added by using append_raw("\n").

### CODE REVIEW

File: include/vcpkg/base/messages.h [hunk 1]
```diff
@@ -74,6 +74,12 @@ namespace vcpkg
             return *this;
         }
 
+        LocalizedString& append_nl()
+        {
+            m_data.append("\n");
+            return *this;
+        }
```
Comment (Thomas1664) [inline include/vcpkg/base/messages.h:81]: There used to be a function that did exactly the same but @BillyONeal removed it in #547.

Comment (BillyONeal) [reply include/vcpkg/base/messages.h:81]: I did remove that, but I think @JavierMatosD owns this space now and if he would like an `appendnl` I'm not strongly opposed; the main thing I wanted to be clear in #547 is that 'our newline character is `\n`, do not contort things to get all newlines to be printed to `\n`'.

File: include/vcpkg/base/messages.h [hunk 2]
```diff
@@ -407,6 +413,7 @@ namespace vcpkg
                     (msg::command_line),
                     "",
                     "'{command_line}' can only add one artifact at a time.");
+    DECLARE_MESSAGE(AddCommandFirstArg, (), "", "The first parameter to add must be 'artifact' or 'port'.");
```
Comment (ras0219-msft) [inline include/vcpkg/base/messages.h:416]: This message is command-specific and should probably be in that command's cpp

File: include/vcpkg/base/messages.h [hunk 3]
```diff
@@ -500,6 +507,7 @@ namespace vcpkg
                     "",
                     "Another installation is in progress on the machine, sleeping 6s before retrying.");
     DECLARE_MESSAGE(AppliedUserIntegration, (), "", "Applied user-wide integration for this vcpkg root.");
+    DECLARE_MESSAGE(ArtifactsOptionIncompatibility, (msg::option), "", "--{option} has no effect on find artifact.");
```
Comment (ras0219-msft) [inline include/vcpkg/base/messages.h:510]: This message looks command specific.

File: include/vcpkg/base/messages.h [hunk 4]
```diff
@@ -795,6 +808,11 @@ namespace vcpkg
                     "expected value after '{value}'.");
     DECLARE_MESSAGE(ExtendedDocumentationAtUrl, (msg::url), "", "Extended documentation available at '{url}'.");
     DECLARE_MESSAGE(FailedToExtract, (msg::path), "", "Failed to extract \"{path}\":");
+    DECLARE_MESSAGE(FailedToFormatMissingFile,
```
Comment (ras0219-msft) [inline include/vcpkg/base/messages.h:811]: This seems very specific to the format command.

**Review by ras0219-msft — COMMENTED** [review]
~This PR adds a lot of command-specific messages to the generic `messages.h` header. I recommend going through each of them and consider moving each one that's only used once into the respective .cpp.~ I see that this is now intended due to https://github.com/microsoft/vcpkg-tool/pull/615. Please disregard this review.

File: include/vcpkg/base/messages.h (hunk 2 above)
Comment (BillyONeal) [reply include/vcpkg/base/messages.h:416]: The current design puts *all* messages here, there are no longer any messages in specific .cpps.

**Review by BillyONeal — APPROVED** [review]

**Pull request merged**